platform-sdk/swirlds-tss/src/main/java/com/swirlds/tss/api/Tss.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.tss.api;
+
+import com.swirlds.signaturescheme.api.SignatureSchema;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.Random;
+
+/**
+ * A Threshold Signature Scheme.
+ * <p>
+ * Contract of TSS:
+ * <ul>
+ *     <li>produce a public key for each share</li>
+ *     <li>give the corresponding secret to the shareholder</li>
+ * </ul>
+ */
+public interface Tss {
```

Comment:
I'm surprised that this and TSS Utils are separate entities when their methods are at the same abstraction layer.  Could the all be on the TSS interface?    If there truly nothing in the TSS implementation that the methods of TSS Utils need?   If that's the case, they could be added here as default implementations.

Reply:
ack, the distinction between these 2 classes isn't super well considered.